docs/Guide.Jest.md

```diff
@@ -19,32 +19,39 @@ You should remove `e2e/mocha.opts`, you no longer need it.
 ### 3. Write a detox setup file
 
 ```js
+// ./jest/setup/
 const detox = require('detox');
 const config = require('../package.json').detox;
 
 // Set the default timeout
 jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000;
 
-beforeAll(async () => {
-  await detox.init(config);
-});
+// setup detox only when running e2e tests
+if (process.argv[2].includes('__e2e__')) {
```

Comment:
This is not needed when we specify the setup files, which I think is a cleaner approach

Reply:
I thought about one file of setup for all environment (including unit tests). But we can run this file specifically for e2e tests as you mention :)